cli/cppcheckexecutor.h

```diff
@@ -48,7 +48,7 @@ class CppCheckExecutor : public ErrorLogger {
     /**
      * Destructor
      */
-    virtual ~CppCheckExecutor();
+    ~CppCheckExecutor() OVERRIDE;
```

Comment:
ok I guess it does not hurt.. but I don't really see the point to use `override` for destructors. :-)